lib/rpc/config.ts

```diff
@@ -57,7 +57,7 @@ export const DEFAULT_SINGLE_PROVIDER_CONFIG: SingleJsonRpcProviderConfig = {
   RECOVER_SCORE_PER_MS: 0.01,
   RECOVER_MAX_WAIT_TIME_TO_ACKNOWLEDGE_IN_MS: 60000,
   ENABLE_DB_SYNC: true,
-  DB_SYNC_INTERVAL_IN_S: 5,
+  DB_SYNC_INTERVAL_IN_S: 10,
```

Comment:
Selecting 10 over 5 feels a bit arbitrary to me. Why not 15s then, to be consistent with evaluation wait period?

Reply:
Yeah it's an arbitrary number. I can change it to 15.